genon/preprocessor/tests/conftest.py

```
@@ -64,3 +64,23 @@ def attachment_processor():
     return mod.DocumentProcessor
+
+
+@pytest.fixture(autouse=True)
+def _stub_vlm_for_unit_tests(request, monkeypatch):
+    """unit 테스트에서는 외부 VLM 호출을 기본 차단한다."""
+    if request.node.get_closest_marker("unit") is None:
+        return
+
+    try:
+        import facade.parser_processor as parser_mod
+    except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)